documentation/modules/rql/pages/user-guide.adoc

```diff
@@ -189,30 +209,36 @@ Escape-char is backslash `\`, possible escape sequences are
 `\"`:: Double Quote
 `\f`:: Formfeed - *DO NOT USE* _this might be removed in future_
 `\b`:: Backspace - *DO NOT USE* _this might be removed in future_
+//TODO ? What do they do, why would I add them to a query?
 
 ==== Times
 
-This parser (and model) support time literals as `OffsetDateTime` represented in a *subset* of _ISO 8601_
-Some less used features of ISO 8601 are not supported - and likely will not in near future:
+This parser (and model) support time literals as `OffsetDateTime` represented in a *subset* of _ISO 8601_.
+Some less used features of ISO 8601 are not supported -- and most likely will not in near future:
 
 * Reduced Precision: giving a partial time `hh` or `hh:mm` is not supported.
-* Fractional hours and minutes (_ISO8601_ states that you can use fractions of the most precise time component (Hour, minute, second).
+* Fractional hours and minutes (_ISO8601_ states that you can use fractions of the most precise time component (hour, minute, second).
 As we only support a fixed precision including seconds this is not supported.
 
-So `2007-12-03T10:15Z` and `2007-12-03T10:15+4:27` are valid _ISO8601_ but *not* valid RQL
+So `2007-12-03T10:15Z` and `2007-12-03T10:15+4:27` are valid _ISO8601_ but *not* valid RQL.
+//TODO ? Like generally not valid for RQL or not valid within this Bosch Semantic Stack RQL project? (Also, just noting that so far it was not explained how to actually set up the RQL project so that it can be used. Or is there nothing to set up because the DTR supports it just natively? (Then why is it an extra open-source project and not just a DTR feature? Can it also be used for other registries or even any other API?))
 
 Valid examples are `2007-12-03T10:15:30.0123Z` and `2007-12-03t10:15:30+04:37`.
 
 <<<
+//TODO ?
 
 === Set comparisons
+//TODO ? Means comparing data SETs? Or to set a comparison? Or that there are some comparisons that have been set by default? Lost here. What about an intro sentence and one or two examples like above?
```

Comment:
array of values, a set of values

Reply:
camparison chapter -- new column in table see below

Reply:
Done -- https://github.com/bci-oss/semanticstack-rql/pull/69/commits/1e458970200ea626548df407c0e4249529893341